docs/themes/coral/scripts/graphqldocs.js

```diff
@@ -9,21 +9,35 @@ const docsScript = fs.readFileSync(
 );
 
 hexo.extend.tag.register('graphqldocs', args => {
-  const filename = path.resolve(hexo.source_dir, args[0]);
-  const introspectionQuery = fs.readFileSync(filename, { encoding: 'utf8' });
+  const schema = args[0];
+  let fetcher = '';
+
+  if (schema.match(/^http[s]?/)) {
+    fetcher = `
+      function fetcher() {
+        return fetch("${schema}").then((result) => result.json()).then((json) => {return {data: json}});
```

Comment:
For broader compatibility, can we include the fetch polyfill from the same cdnjs? See https://cdnjs.com/libraries/fetch.